### PR TITLE
User Reservation: Add Map View & Update Address Form

### PR DESCRIPTION
**Pull request by clee4037**
- Integrated map view
- Connected front and backend
- Using geocode endpoint to convert address to long and lat coordinates
- Removed country field from address from
- Added 2 letter state dropdown to address form
- Added route to user reservation page

### CODE REVIEW

File: src/App.jsx
Comment (stevkim) [inline src/App.jsx:1]: XD the pokemon path lol

File: src/components/user-reservation/MapView.jsx
Comment (stevkim) [inline src/components/user-reservation/MapView.jsx:1]: Super cool how you got the maps going

**Review by stevkim — APPROVED** [review]
Everything looks good to me!
Quick note - it looks like getGarageData/getGeoCoordinates are the same functions?
Also, idk if its a Supabase thing but you might not need to JSON.parse the data? *would need to test this

**Comment by clee4037** [comment]
@stevkim 
Good catch -- getGarageData is WIP!
I tested and JSON.parse is required.

File: src/containers/UserReservation.jsx
Comment (bzw24) [inline src/containers/UserReservation.jsx:1]: Edit to: "../components/user-reservation/Confirmation.jsx" to eliminate compilation error (your component is named 'Confirmation.jsx'

**Review by bzw24 — CHANGES_REQUESTED** [review]
Made note on one change for compilation issues. Otherwise module is looking good. I like the separations of concerns by separating axios calls under the utils folder.

Under your UserReservation.jsx, edit the path to: "../components/user-reservation/Confirmation.jsx" to eliminate compilation error (your component is named 'Confirmation.jsx' not 'confirmatin.jsx' (capitalization)

**Review by bzw24 — APPROVED** [review]

**Pull request merged**